packages/next-ui/Theme/MuiSnackbar.ts

```diff
@@ -22,8 +22,10 @@ export const MuiSnackbar: SnackbarVariants = [
           },
         },
       '&.MuiSnackbar-anchorOriginBottomCenter': {
-        left: 0,
-        right: 0,
+        [theme.breakpoints.up('md')]: {
+          left: theme.page.horizontal,
+          right: theme.page.horizontal,
+        },
```

Comment:
We always lived without this, still necessary?

Reply:
![Screenshot 2023-11-15 at 16 04 41](https://github.com/graphcommerce-org/graphcommerce/assets/7457340/ccc18319-220c-4964-b404-80a6e3ba06eb)
Without this, snackbars with content >= viewport width will stick to the sides of the viewport. Looks a bit odd.